Trappist/src/Promact.Trappist.Web/Migrations/20170427123317_Regex-Change-For-Testname.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System;
+using System.Collections.Generic;
+using Microsoft.EntityFrameworkCore.Migrations;
+
+namespace Promact.Trappist.Web.Migrations
+{
+    public partial class RegexChangeForTestname : Migration
+    {
+        protected override void Up(MigrationBuilder migrationBuilder)
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Add a nested comment explaining why this method is empty, throw a 'NotSupportedException' or complete the implementation. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://server12:9005/coding_rules#rule_key=csharpsquid%3AS1186)